docs/src/app/components/pages/components/Card/ExampleControlled.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import Card from 'material-ui/lib/card/card';
+import CardActions from 'material-ui/lib/card/card-actions';
+import CardHeader from 'material-ui/lib/card/card-header';
+import CardMedia from 'material-ui/lib/card/card-media';
+import CardTitle from 'material-ui/lib/card/card-title';
+import FlatButton from 'material-ui/lib/flat-button';
+import CardText from 'material-ui/lib/card/card-text';
+
+
+export default class CardExampleControlled extends React.Component {
+
+  constructor(props, context) {
+    super(props, context);
+    this.state = {
+      expanded: false,
+    };
+  }
+
+  onExpandChange = (expanded) => {
```

Comment:
`handleExpandChange` to stay coherent with the code base (https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-handler-names.md)